.github/workflows/test.yml

```diff
@@ -0,0 +1,22 @@
+name: Test
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
```

Comment:
We may add concurrent settings: https://github.com/api7/cloud/blob/main/.github/workflows/e2e-tests.yaml#L10.

Reply:
Fixed.